src/System.Collections/tests/Performance/Configurations.props

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netcoreapp;
+      netstandard;
```

Comment:
Why are we changing the perf tests configurations to netstandard?

Reply:
Discussed offline. Promoting them to netstandard allows running in different configurations.